### ISSUE TITLE
Allow to customize the application icon per workspace

### ISSUE THREAD

**Issue by skeddles** [microsoft/vscode#164157]
when you have multiple windows open, the icon is all the same, and the name is just a filename, so it's hard to tell which as which without clicking between them each time. 

we should be able to supply a custom icon that shows in the task bar.

the current nightmare:
![image](https://user-images.githubusercontent.com/2721332/197021135-c28023d1-4061-433d-b3ef-798efb2792a8.png)

the dream:
![image](https://user-images.githubusercontent.com/2721332/197021596-c5a4468a-6ad2-4ed2-be33-9947b952b4ca.png)

previously suggested here: https://github.com/microsoft/vscode/issues/99415 and https://github.com/microsoft/vscode/issues/84888

### IF YOU AGREE PLEASE CLICK 👍 BELOW

**Comment by skeddles** [comment]
@Awbmilne, @MKe090, @5dao, @QuentinDanjou, @ClintOxx, @cejaramillof, @luoshan654, @heartacker, @idanpa, @theS1LV3R @akrymskiy, @sayore, @SarrenRool, @Pidi64, @whizsid, @TrueMattreal, @mfearby, @patrick984, @gjsjohnmurray have previously voted for this

**Comment by sayore** [comment]
YES PLEASE make this a reality

**Comment by adamPhoebe** [comment]
This is such an obvious feature, I can't believe they haven't implemented it yet. VSCode, oh my god please add this.

**Comment by skeddles** [comment]
We reached 20 thumbs, thanks everyone. Hopefully this means now it will be considered for real and not closed.

**Comment by skeddles** [comment]
@bpasero  just making sure you're aware

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_smiling_face: This feature request received a sufficient number of community upvotes and we moved it to our backlog. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!

**Comment by sayore** [comment]
you cant be serious

**Comment by Awbmilne** [comment]
This would be an incredibly useful function for developers that work on multiple repositories in parallel. Yeah, sure, you can have a workspace that includes multiple repos, but unless you use it that way all the time, it becomes a huge pain. I _really_ think this should be reconsidered.

**Comment by skeddles** [comment]
> Please see #167999 for our plan this month to clean up and "groom" all opened issues. This issue was closed as part of that initiative.

if there's too many suggestions to keep track of them, maybe you can convince them to switch to a more appropriate method of collecting suggestions, like a Canny.io board. That would let people submit and vote on ideas, you wouldn't have to delete ones that get less than 20, or worry about duplicates from people reopening threads that didnt get enough votes (like this one). You wouldn't have to close any, you can just leave them open until they can be worked on. I think it would be better for both us and you.
